internal/rset/db.go

```diff
@@ -0,0 +1,26 @@
+package rset
+
+import (
+	"database/sql"
+	"github.com/nalgeon/redka/internal/sqlx"
+)
+
+// DB is a database-backed set repository.
+// A set is a field-value map associated with a key.
+// Use the set repository to work with individual hashmaps
+// and their fields.
```

Comment:
Not true for sets. It's OK to copy-paste, but please remember to tailor the code/comments to the feature you're working on.

Reply:
Yes, I know. This PR is not ready yet. I am going to update it. 👍🏻